src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.html

```diff
@@ -140,17 +146,17 @@ <h5 class="modal-title"><i class="fas fa-exclamation-circle"></i> Confirm archiv
     </ng-template>
 
     <div class="card-body padding-0" *ngIf="courseTabModel.isTabExpanded">
-      <tm-sessions-table [sessionsTableRowModels]="courseTabModel.sessionsTableRowModels" [sessionsTableRowModelsSortBy]="courseTabModel.sessionsTableRowModelsSortBy"
-                         [sessionsTableRowModelsSortOrder]="courseTabModel.sessionsTableRowModelsSortOrder" [courseCandidates]="courseCandidates"
-                         [headerColorScheme]="SessionsTableHeaderColorScheme.WHITE"
-                         [columnsToShow]="[ SessionsTableColumn.START_DATE, SessionsTableColumn.END_DATE ]"
-                         (sortSessionsTableRowModelsEvent)="sortSessionsTableRowModelsEvent(idx, $event)"
-                         (loadResponseRateEvent)="loadResponseRateEventHandler(idx, $event)" (editSessionEvent)="editSessionEventHandler(idx, $event)"
-                         (moveSessionToRecycleBinEvent)="moveSessionToRecycleBinEventHandler(idx, $event)" (copySessionEvent)="copySessionEventHandler(idx, $event)"
-                         (submitSessionAsInstructorEvent)="submitSessionAsInstructorEventHandler(idx, $event)" (viewSessionResultEvent)="viewSessionResultEventHandler(idx, $event)"
-                         (publishSessionEvent)="publishSessionEventHandler(idx, $event)" (unpublishSessionEvent)="unpublishSessionEventHandler(idx, $event)"
-                         (resendResultsLinkToStudentsEvent)="resendResultsLinkToStudentsEventHandler(idx, $event)"
-                         (sendRemindersToStudentsEvent)="sendRemindersToStudentsEventHandler(idx, $event)"></tm-sessions-table>
+      <tm-sessions-table *ngIf="courseTabModel.sessionsTableRowModels.length > 0; else noSessionMessage"
```

Comment:
I think you should change the component `tm-sessions-table` itself rather than this page...